### ISSUE TITLE
InstructorFeedbackEditPageUiTest: fix failure against live server

### ISSUE THREAD

**Issue by damithc** [TEAMMATES/teammates#7937]
v5.110
Possibly related to how visibility messages take longer to appear when testing against live server
```
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:327)
	at org.openqa.selenium.remote.RemoteWebElement.findElement(RemoteWebElement.java:198)
	at org.openqa.selenium.remote.RemoteWebElement.findElementByCssSelector(RemoteWebElement.java:287)
	at org.openqa.selenium.By$ByCssSelector.findElement(By.java:430)
	at org.openqa.selenium.remote.RemoteWebElement.findElement(RemoteWebElement.java:194)
	at teammates.test.pageobjects.InstructorFeedbackEditPage.verifyVisibilityMessageIsDisplayed(InstructorFeedbackEditPage.java:1304)
	at teammates.test.cases.browsertests.InstructorFeedbackEditPageUiTest.testChangeFeedbackRecipient(InstructorFeedbackEditPageUiTest.java:680)
	at teammates.test.cases.browsertests.InstructorFeedbackEditPageUiTest.testGeneralQuestionOperations(InstructorFeedbackEditPageUiTest.java:98)
	at teammates.test.cases.browsertests.InstructorFeedbackEditPageUiTest.allTests(InstructorFeedbackEditPageUiTest.java:73)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
```

**Comment by damithc** [comment]
Did not fail at v6.3

**Comment by tshradheya** [comment]
Does this test still fail? 
I think the cause of the test failing was fixed in #7452

**Comment by damithc** [comment]
Didn't fail in recent times. Closing.
